packages/common/src/types/visualizations.ts

```diff
@@ -0,0 +1,31 @@
+export type BarChartConfig = {
+    metadata: {
+        version: number;
+    };
+    // TODO: There was some back and forth about whether this should be 'defaultType'
+    // I'm not sure
+    type: 'barChart';
```

Comment:
we can remove this - not needed for MVP

Reply:
Cause we don't know how we want to do this yet?

Reply:
exactly 👍